### DISCUSSION TITLE
Updates filling up disk

### DISCUSSION THREAD

**Discussion by adispy** [uyuni-project/uyuni#9028] [General]
Hi everyone,
Just installed Uyuni, trying to learn the software, but after I put the Ubuntu repos in Software > Repositories and do a sync, my drive gets filled really fast. Looks like Uyuni is downloading the entire Ubuntu repo.
Is there any way to just get the patch catalog and download only the approved updates? Sort like Microsoft WSUS.

Thanks,

**Comment by mcalmer** [comment]
Yes, Uyuni fully mirror the repositories. There is a Google Summer of Code project running to make a Lazy Sync possible.
Check the last Community Hours Video to get more information
